src/devices_models/devices/AC_branches.jl

```diff
@@ -59,32 +59,6 @@ function branch_rate_bounds!(
     return
 end
 
-function branch_rate_bounds!(
```

Comment:
use this function definition in the function line 88 we don't need the D and S variables